components/server/src/main/java/com/hotels/styx/server/netty/connectors/HttpResponseWriter.java

```diff
@@ -71,7 +72,9 @@
                 }
             });
 
-            Subscription subscriber = toRxObservable(response.body()).subscribe(new Subscriber<ByteBuf>() {
+            // TODO: Mikko
```

Comment:
TODOs should be converted to issues.

Reply:
Removed these todos. They are unnecessary.